tests/acceptance/features/apiSharingNg1/sharedWithMe.feature

```diff
@@ -5268,3 +5268,113 @@ Feature: an user gets the resources shared to them
+    Given using spaces DAV path
+    And the administrator has enabled the permissions role "Denied"
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "NewSpace" with the default quota using the Graph API
+    And user "Alice" has created a folder "FolderToShare" in space "NewSpace"
+    And user "Alice" has uploaded a file inside space "NewSpace" with content "hello world" to "FolderToShare/lorem.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | FolderToShare      |
+      | space           | NewSpace           |
+      | sharee          | Brian              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
+    And user "Alice" has updated the last resource share with the following properties:
+      | permissionsRole | Denied        |
+      | space           | NewSpace      |
+      | resource        | FolderToShare |
+    When user "Brian" lists the shares shared with him using the Graph API
+    Then the HTTP status code should be "200"
+    And user "Brian" should not have a share "FolderToShare" shared by user "Alice" from space "NewSpace"
+    And user "Brian" should not be able to download file "FolderToShare/lorem.txt" from space "Shares"
+    Examples:
+      | permissions-role |
+      | Editor           |
+      | Viewer           |
+      | Uploader         |
+
+  @env-config
+  Scenario Outline: check share access after updating the permission role of a shared folder from Denied to other roles (Project Space)
+    Given using spaces DAV path
+    And the administrator has enabled the permissions role "Denied"
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "NewSpace" with the default quota using the Graph API
+    And user "Alice" has created a folder "FolderToShare" in space "NewSpace"
+    And user "Alice" has uploaded a file inside space "NewSpace" with content "hello world" to "FolderToShare/lorem.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | FolderToShare |
+      | space           | NewSpace      |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | Denied        |
+    And user "Alice" has updated the last resource share with the following properties:
+      | permissionsRole | <permissions-role> |
+      | space           | NewSpace           |
+      | resource        | FolderToShare      |
+    When user "Brian" lists the shares shared with him using the Graph API
+    Then the HTTP status code should be "200"
+    And user "Brian" should have a share "FolderToShare" shared by user "Alice" from space "NewSpace"
+    And user "Brian" should be able to download file "FolderToShare/lorem.txt" from space "Shares"
```

Comment:
```suggestion
```

Reply:
:+1: the content check step below also checks the download.